build/js/tests.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
+
+// This file is just a file that helped me in testing password and mail without having to manually write the same stuff over and over again
+// and yeah - I need to learn some tool for unit tests
+
+var primitiveTest = function primitiveTest(tested, against) {
```

Comment:
It got added by while babelizing. Here's how it looked like before:

`const primitiveTest = function(tested, against) {
    return tested === against;
};`